source/mainnet/net/guides/recover-wallet.rst

```diff
@@ -22,11 +22,23 @@ In case you get a new device or need to restore your existing device, you can re
 
     #. Select whether to recover your wallet in Mainnet or Testnet.
 
+    If recovery is successful, you will see a screen similar to below. If recovery is only partial, you can try to recover using the **Restore IDs and accounts option** in Wallet settings.
+
+    .. image:: ../images/browser-wallet/recovery-success-bw.png
+        :width: 25%
+
+    .. image:: ../images/browser-wallet/wallet-settings.png
+        :width: 25%
+
     .. Note::
 
         You cannot recover a wallet created in Mainnet in Testnet, and vice versa. Once you have recovered on the correct network, you can :ref:`switch between mainnet and testnet<switch-network>`.
         If you use both networks, you can recover on the other net after setup, through the settings page.
 
+    .. Warning::
+
+        If you had previously had tokens in your wallet, these will not be recovered. You must add the tokens again.
```

Comment:
Is this phrasing scary or is it just me? Can sound a bit like you have lost your tokens, if you don't know any better?

Maybe something like `If you had previously configured your wallet to be able to view selected tokens, this will not be recovered. You must add the tokens again`

Reply:
`... these will not be recovered automatically recovered. You must add the tokens to view them again.`
Maybe?